### DISCUSSION TITLE
dollar sign in string substitution variable

### DISCUSSION THREAD

**Discussion by phsaucier** [google/zx#773] [Help needed] [answered]
How can I get this .mjs to substitute correctly the password containing a dollar sign ? 
```
import 'zx'
import { $, argv, cd, fs, os } from 'zx'

const login = 'someUser'
const pass = '"pwdWithA$"'

await $`someDbCommandLine user=${login} password=${pass}`
```

**Actual result:** 

$ someDbCommandLine user=someUser password=$'"pwdWithA$"'

**Expected :** 

$ someDbCommandLine user=someUser password="pwdWithA$"

**Comment by antongolub** [comment]
As a workaround for now:
```ts
${quote: v => v}`cmd "at" "your own" 'risk'`
```

**Comment by antonmedv** [comment, marked as answer]
Drop quotes in password:
```diff
const login = 'someUser'
-const pass = '"pwdWithA$"'
+const pass = 'pwdWithA$'

await $`someDbCommandLine user=${login} password=${pass}`
```

Read more about https://google.github.io/zx/quotes
